### ISSUE TITLE
Printer stops in the middle of printing

### ISSUE THREAD

**Issue by Darkstars123** [Lunatrius/Schematica#99]
So recently while using the printer on my server, i found out that sometimes when i printing builds, and it fails to place a block, it just stops printing for that block. The server has spigot, but no anticheat/build. I changed the config a little, but i dont want to have to go back to defaults because it builds too slow. (Configuration settings below) The buildings i try to build with the printer use a lot of slabs. Please reply soon as this gets very annoying

These are the config settings i have:
Configuration file

```
~CONFIG_VERSION: 1

debug {
    # Dump all block states on startup.
    B:dumpBlockList=false

    # Display extra information on the debug screen (F3).
    B:showDebugInfo=true
}


general {
    # Extra blocks to consider as air for the schematic renderer.
    S:extraAirBlocks <
     >

    # Schematic directory.
    S:schematicDirectory=schematics

    # Default sort type for the material list.
    S:sortType=SIZE_DESC
}


printer {
    # The printer will destroy blocks (creative mode only).
    B:destroyBlocks=false

    # Destroy all blocks that can be destroyed in one tick.
    B:destroyInstantly=false

    # Place blocks only if there is an adjacent block next to them.
    B:placeAdjacent=true

    # Delay between placement attempts (in ticks).
    I:placeDelay=0

    # Place all blocks that can be placed in one tick.
    B:placeInstantly=false

    # Timeout before re-trying failed blocks.
    I:timeout=0

    swapslots {
        # Allow the printer to use this hotbar slot.
        B:swapSlot0=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot1=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot2=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot3=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot4=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot5=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot6=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot7=true

        # Allow the printer to use this hotbar slot.
        B:swapSlot8=false
    }

}


render {
    # Alpha value used when rendering the schematic (1.0 = opaque, 0.5 = half transparent, 0.0 = transparent).
    D:alpha=0.5

    # Enable transparent textures.
    B:alphaEnabled=false

    # Delta value used for highlighting (if you experience z-fighting increase this).
    D:blockDelta=0.005

    # Highlight invalid placed blocks and to be placed blocks.
    B:highlight=true

    # Highlight blocks that should be air.
    B:highlightAir=true

    # Schematic render distance.
    I:renderDistance=8
}


server {
    # Allow players to load schematics.
    B:loadEnabled=true

    # Amount of storage provided per-player for schematics on the server.
    I:playerQuotaKilobytes=8192

    # Allow players to use the printer.
    B:printerEnabled=true

    # Allow players to save schematics.
    B:saveEnabled=true
}
```

**Comment by Lunatrius** [comment]
Don't set the placement delay bellow 1. It's a bug and will be addressed in the following few days.

**Comment by Darkstars123** [comment]
@Lunatrius Sorry, but does "addressed" mean you'll fix it and be able to set it to 0 without issues? And thanks so much for the quick reply.

**Comment by Lunatrius** [comment]
It means that it will be changed/fixed.

**Comment by Lunatrius** [comment]
Fixed in af0a774216ae01c8798dabebbb33a72276ef327f. Turns out it was just a debugging thing that accidentally slipped through...
